physionet-django/project/test_views.py

```diff
@@ -1233,12 +1233,12 @@ def test_invalid_access(self):
             self.assertEqual(response.status_code, HTTPStatus.FORBIDDEN)
 
         # awaiting editor decision
-        self.client.login(username='tpollard@mit.edu', password='Tester11!')
+        self.client.login(username='rgmark@mit.edu', password='Tester11!')
```

Comment:
Just noticed your comment on Slack ("the test was failing because one of the test user had admin permission and had direct access to the project edit"). Makes sense!

Reply:
Hi, So the user tpollard previously had is_admin permission, and after we added the permission for console. I had updated the demo-user.json(fixtures) to add the tpollard user to Admin group.

To followup with the new permission, i update the views in PR #1723 to fix the file upload process(which was also using is_admin flag), for that change i had created a generic function is_editable_by which checks if the current user should be allowed to edit the project.(and that function lets anyone with `project.change_project` permission change the project anytime).

So the user tpollard had the `project.change_project` permission, and the test was failing as the test was trying to make sure that editor should not be able to make changes to project(but because tpollard had the `project.change_project` permission, he could edit the project and the test failed)